### PR TITLE
added hint about Java 11 to new Xtext project wizard.

### PR DESCRIPTION
**Pull request by cdietrich**
added hint about Java 11 to new Xtext project wizard.
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/WizardNewXtextProjectCreationPage.java [hunk 1]
```diff
@@ -314,4 +318,12 @@ public JavaVersion getJavaVersion() {
 		JavaVersion version = JavaVersion.fromBree(selected);
 		return version;
 	}
+	
+	@Override
+	public void setVisible(boolean visible) {
+		super.setVisible(visible);
+		if (visible) {
+			validatePage();
```
Comment (kthoms) [inline org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/WizardNewXtextProjectCreationPage.java:326]: Why do this is setVisible? Looks like an unexpected side effect here. Is there another place better suited to call it?
I guess the intention is here to validate when the page becomes active.

Comment (cdietrich) [reply org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/WizardNewXtextProjectCreationPage.java:326]: I don’t know the correct place to do that on there is nobody available to tell which one is better

Comment (kthoms) [reply org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/WizardNewXtextProjectCreationPage.java:326]: Adding `validatePage()` to `createControl` seems to do the same job. It is just the initial validation that you want to trigger, all further validation is already handled by editing the page.

Comment (cdietrich) [reply org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/WizardNewXtextProjectCreationPage.java:326]: thisis about page flipping

**Comment by cdietrich** [comment]
obsolete for now

**Pull request closed without merging**